apps/doji-backend/src/session/session.dto.ts

```diff
@@ -91,3 +95,25 @@ export class SessionDTO implements ISession {
   @ApiProperty()
   service: IService
 }
+
+export class SessionInformationDTO implements ISessionInformationDTO {
+  @ApiProperty()
+  id: number
+  @ApiProperty()
+  reviews: IPublicSessionReviewDTO[]
+  @ApiProperty()
+  reviewStat: ReviewStatDTO
+}
+
+export class PublicSessionReviewDTO implements IPublicSessionReviewDTO {
+  @ApiProperty()
+  id: number
+  @ApiProperty()
+  rating: number
+  @ApiProperty()
+  authorName: string
+  @ApiProperty()
+  content: string
+  @ApiProperty()
+  createdAt: Date
```

Comment:
spacing

Reply:
I mean 
```suggestion
  @ApiProperty()
  id: number
  
  @ApiProperty()
  reviews: IPublicSessionReviewDTO[]
  
  @ApiProperty()
  reviewStat: ReviewStatDTO
}

export class PublicSessionReviewDTO implements IPublicSessionReviewDTO {
  @ApiProperty()
  id: number
  
  @ApiProperty()
  rating: number
  
  @ApiProperty()
  authorName: string
  
  @ApiProperty()
  content: string
  
  @ApiProperty()
  createdAt: Date
```

Reply:
Oh I forgot. name the DTO end with `Response` or `Request` like 

`PublicSessionReviewResponseDTO `

Reply:
And should we use `class-validator` on this?

Reply:
- Ok
- No need. It is output only